src/datetime/now.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * To return a DateTime data type value corresponding to the current date and time.
+ * @return Date
+ */
+
+export const now = (): Date => {
+  return new Date();
```

Comment:
Shouldn't this function consider the current timezone?

Reply:
you are right !!, fixed